### PR TITLE
Fix vyper installation

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

**Comment by lyubenkov** [comment]
Green vyper build http://neon-test-allure.s3-website.eu-central-1.amazonaws.com/compiler_compatibility/devnet/NDEV-2772/50/#suites/a53e7bb4f8308b94fb95c5f5adcf83f1

File: contracts/vyper/Erc20.vy [hunk 1]
```diff
@@ -24,7 +24,7 @@ totalSupply: public(uint256)
 minter: address
 
 
-@external
+@deploy
```
Comment (gigimon) [inline contracts/vyper/Erc20.vy:27]: Did you check why this changes need? Maybe vyper version is newer?

**Review by gigimon — APPROVED** [review]

File: contracts/vyper/Erc20.vy (hunk 1 above)
Comment (lyubenkov) [reply contracts/vyper/Erc20.vy:27]: Yes, it's due to the newer versions, we're testing now with beta vyper, such as 4beta4 (Kristina will create new issue to handle this gracefully).
https://pypi.org/project/vyper/#history

**Review by kristinaNikolaevaa — APPROVED** [review]
